src/pages/Main.js

```diff
@@ -1,3 +1,3 @@
 export default function Main() {
-    return <h1>Coming Soon!</h1>
+    return <h1>Coming Soon! :) </h1>
```

Comment:
Request(nb): Add another HTML element and style it to use the Monserrat font so we can see that too. It will be fine to use inline styles for now, since we'll make classes when we get the designs.

Reply:
Added constructed text with the Montserra font.